docs/design/specs/Ecma-335-Augments.md

```diff
@@ -439,6 +443,8 @@ The algorithm is amended as follows:
 
 **Section** "III.2.1 constrained. prefix" the paragraph starting with "This last case can only occur when method was defined on `System.Object`, `System.ValueType`, or `System.Enum`" is extended to also cover default interface method implementation. In the case the interface method implementation is provided by an interface, the implicit boxing becomes _observable_ to the program.
 
+**Section** "III.3.41 ldftn" is extended to allow throwing `AmbiguousImplementationException` if the implementation of the static virtual interface method resolves at runtime to more than one default interface method. It's also extended to specify throwing `EntryPointNotFoundException` if the default interface implementation is abstract.
+
```

Comment:
There should also be a clause about how "call" is extended to allow throwing `AmbiguousImplementationException`/`EntryPointNotFoundException`. It should be just like the rest of these.

Reply:
Thanks for pointing that out, added in 2nd commit.